packages/http-client/src/index.d.ts

```diff
@@ -4,17 +4,20 @@ import type {
   UserProp,
 } from "@webstudio-is/react-sdk";
 
-export type IncludeTypes = "tree" | "props" | "breakpoints" | "pages";
-export type Includes<T> = { [include in IncludeTypes]: T };
 export type Page = {
   id: string;
   root: [string, string];
 };
-export type Project = {
-  tree: BaseInstance & { errors: string };
-  props: Array<UserProp> & { errors: string };
-  breakpoints: Array<Breakpoint> & {
-    errors: string;
-  };
-  pages: Record<string, unknown> | null;
-};
+export type Project =
+  | {
+      pages: {
+        [string]: {
+          page: Page;
+          tree: BaseInstance;
+          props: Array<UserProp> | [];
+          breakpoints: Array<Breakpoint> | null;
+          css: string;
+        };
+      };
+    }
+  | Error;
```

Comment:
`Error` seem wrong here:

 1. We shouldn't use native Error type probably unless it's in context of try..catch or something
 2. Same as with `undefined` before, this is wrong semantically: a error is not a `Project`